pkg/injector/pod_patch.go

```diff
@@ -472,6 +478,7 @@ func getSidecarContainer(annotations map[string]string, id, daprSidecarImage, na
 		log.Warn(err)
 	}
 
+	sslEnabled := appSSLEnabled(annotations)
```

Comment:
Can you please add the unit-test for `getSidecarContainer` ?

Reply:
Added.